### PR TITLE
Gamelab: World.seconds block

### PR DESCRIPTION
**Pull request by caleybrock**
Seconds block returns the number of seconds since the user started running the program.

### CODE REVIEW

File: apps/test/unit/gamelab/GameLabGameTest.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+describe('seconds', function () {
+  it('returns time passed in seconds', function () {
+    stub(Date.prototype, 'getTime');
+    // Time at the start of the program.
+    Date.prototype.getTime.onCall(0).returns(5300);
+    // Time when seconds gets called.
+    Date.prototype.getTime.onCall(1).returns(10000);
+
+    let gameLabGame = new GameLabGame({});
+    console.log("game" + gameLabGame.startTime);
```
Comment (Bjvanminnen) [inline apps/test/unit/gamelab/GameLabGameTest.js]: console.log left here intentionally?

File: apps/src/gamelab/GameLabGame.js [hunk 2]
```diff
@@ -37,4 +40,13 @@ Object.defineProperty(GameLabGame.prototype, 'frameRate', {
...
 
+Object.defineProperty(GameLabGame.prototype, 'seconds', {
+  enumerable: true,
+  get: function () {
+    const currentDate = new Date();
+    const currentTime = currentDate.getTime();
+    return Math.round((currentTime - this.startTime) / 1000);
+  }
+});
+
```
Comment (Bjvanminnen) [inline apps/src/gamelab/GameLabGame.js:51]: Should this class/file become GameLabWorld instead at some point?

Comment (caleybrock) [reply apps/src/gamelab/GameLabGame.js:51]: Yes, but I think it's probably best to do in a separate PR. I'll add to my list for the sprint.

File: apps/test/unit/gamelab/GameLabGameTest.js [hunk 3]
```diff
@@ -12,5 +12,6 @@ describe('seconds', function () {
 
     let gameLabGame = new GameLabGame({});
     expect(gameLabGame.seconds).to.equal(5);
+    Date.prototype.getTime.restore();
```
Comment (Bjvanminnen) [inline apps/test/unit/gamelab/GameLabGameTest.js]: :tada:

**Pull request merged**